Pablo-R2-book-updates-(CU-8676z6fzk) (#2766)
ComposableFi/composable: docs/docs/products/pablo/launch-pools.md

# Liquidity Pools on Launch

On launch, Pablo will be the only way of swapping for PICA through dual asset constant product pools, 
based on the BalancerFi weighted math implementation of (x * y = K) for:

- KSM/USDT
- PICA/USDT
- PICA/KSM

:::note

Pablo’s native token, PBLO, is set for launch in late Q1 2023.
:::
Instead, liquidity providers(LPs) directly benefit from the success and adoption of Pablo.
LPs are provided LP-tokens relative to the funds they add to a liquidity pool on Pablo.
A percentage of the total buy/sell order and swap fees are distributed to liquidity providers.
Then the accumulated fees are added back to the pool, effectively resulting in a pro-rata redistribution to LPs
based on the LP-token share they redeem and the total value of the pool.

> If a user adds/removes liquidity from a pool, creating a high degree of liquidity imbalance,
> the standard 0.3% fee applies.

Author: ```suggestion
If a user provides liquidity in only one asset to a pool, they are charged the pool fee rate for creating an imbalance.
```

# Liquidity Pools on Launch

On launch, Pablo will be the only way of swapping for PICA through dual asset constant product pools, 
based on the BalancerFi weighted math implementation of (x * y = K) for:

- KSM/USDT
- PICA/USDT
- PICA/KSM

:::note

Pablo’s native token, PBLO, is set for launch in late Q1 2023.
During this time, the entire protocol will be governed by PICA.
After the launch of fNFTs on Picasso, governance will solely be controlled by xPICA holders until the launch of PBLO.
:::

> Unlike other DEXs, Pablo does not rely on a highly inflationary reward token to incentivize users.

Reviewer: don't say this

> During this time, the entire protocol will be governed by PICA.
> After the launch of fNFTs on Picasso, governance will solely be controlled by xPICA holders until the launch of PBLO.
> :::
> 
> Unlike other DEXs, Pablo does not rely on a highly inflationary reward token to incentivize users.

Author: ```suggestion
:::
```

Instead, liquidity providers(LPs) directly benefit from the success and adoption of Pablo.
LPs are provided LP-tokens relative to the funds they add to a liquidity pool on Pablo.

> A percentage of the total buy/sell order and swap fees are distributed to liquidity providers who bond their LP-tokens.

Reviewer: who provide liquidity - no bonding involved

> A percentage of the total buy/sell order and swap fees are distributed to liquidity providers who bond their LP-tokens.

Author: ```suggestion
A percentage of the total buy/sell order and swap fees are distributed to liquidity providers.
```

Then the accumulated fees are added back to the pool, effectively resulting in a pro-rata redistribution to LPs
based on the LP-token share they redeem and the total value of the pool.
If a user adds/removes liquidity from a pool, creating a high degree of liquidity imbalance,

> the standard 0.3% fee applies.

Reviewer: fee is only for swappiing. There is no fee for adding/removing liquidity

Author: According to the comments on the original doc, there is a specific case for add_liquidity for critical imbalance.
@PoisonPhang @benluelo can you confirm or correct this?

Author: Edited with input from @benluelo. Thank you.
